CppCoreGuidelines.md

```diff
@@ -12694,6 +12694,16 @@ Multiple case labels of a single statement is OK:
         break;
     }
 
+Return statements in a case label are also OK:
```

Comment:
If a smaller version is wanted that could also be integrated into the previous example by changing that to `Multiple case labels of a single statement is OK, as is a return statement instead of break:` and `return do_something(x);`